gen_sign_cert.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+if [ -z "$1" ]; then
+	echo "Usage: $0 <common name>"
+	exit
+fi
+
+cn=$1
+echo "Generating cert for common name = " ${cn}
+openssl genrsa -out test.key 2048
+
+openssl req -new -text \
+          -extensions v3_req \
+          -addext "subjectAltName = DNS:${cn}" \
+          -subj "/C=US/O=ovnkubernetes/OU=kind/CN=${cn}" \
+          -key test.key \
+          -out test.csr
+
+csr_64=$(cat test.csr | base64 | tr -d "\n")
+
+cat <<EOF | oc apply -f -
```

Comment:
`cat` is usually never needed

`<<EOF oc apply -f -`

Reply:
ok

Reply:
it doesnt work like that?